examples/persistence-storage/compose.yml

```diff
@@ -0,0 +1,58 @@
+services:
+  db:
+    image: postgres:16-alpine
```

Comment:
tag latest?

Reply:
looked at other places, we usually pin a version -- I think we should stick to that rule here, too